openpype/hosts/maya/api/lib.py

```diff
@@ -2622,10 +2624,14 @@ def _get_attrs(node):
             for attr in transform_attrs:
                 cmds.setAttr('{0}.{1}'.format(new_node, attr), lock=False)
 
+            scale_keys = _remove_scale_keys(node)
+
             # Constraints
             delete_bin.extend(cmds.parentConstraint(node, new_node, mo=False))
             delete_bin.extend(cmds.scaleConstraint(node, new_node, mo=False))
 
+            _set_scale_keys(scale_keys)
+
```

Comment:
```suggestion
                plug = "{}.{}".format(new_node, attr)
                cmds.setAttr(plug, lock=False)
                disconnect_inputs(plug)

            # Constraints
            delete_bin.extend(cmds.parentConstraint(node, new_node, mo=False))
            delete_bin.extend(cmds.scaleConstraint(node, new_node, mo=False))

```

This should avoid the issues and do what originally was intended - which is to take world-space scale into account instead of transfering scale keys 1-to-1.